packages/widgets/bettinggame/src/components/TournamentWidget.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types'
+import EntityDetailApp from 'tocco-entity-detail/src/main'
+
+const TournamentWidget = props => {
+  const {tournamentKey, formBase} = props
```

Comment:
nitpick: align with our coding guidelines

https://docs.tocco.ch/framework/client/code-styleguide.html#react-components
- destruct props in function parameter
```
const TournamentWidget = ({tournamentKey, formBase}) => {}
```

- return component immediately
```
const TournamentWidget = ({...}) => (<EntityDetailApp ... />)
```
instead
```
const TournamentWidget = ({...}) => {
    return (<EntityDetailApp ... />)
}
```